template.tpl

```diff
@@ -342,6 +342,43 @@ ___TEMPLATE_PARAMETERS___
       }
     ]
   },
+  {
+    "type": "GROUP",
+    "name": "googleAdsEnhancedConversionsGroup",
+    "displayName": "Enhanced Conversions",
+    "groupStyle": "ZIPPY_CLOSED",
+    "enablingConditions": [
+      {
+        "paramName": "tagType",
+        "type": "EQUALS",
+        "paramValue": "googleAdsEvent"
+      }
+    ],
+    "subParams": [
+      {
+        "type": "CHECKBOX",
+        "name": "googleAdsEnhancedConversionsCheckbox",
+        "checkboxText": "Include user-provided data from your website",
+        "simpleValueType": true,
+        "help": "Improve measurement and get more insights with data people provide to your website. You will need to agree to the user-provided data terms and policies in your Google Ads account first.",
+      },
+      {
+        "type": "SELECT",
+        "name": "googleAdsEnhancedConversionsUserDataVariable",
```

Comment:
It might be good to include helptext here saying that the variable selected should be of type ` User-Provided Data`

Reply:
Good idea. I just copy pasted the original help text lol